grpc/src/main/java/com/linecorp/armeria/internal/common/grpc/GrpcStatusUtil.java

```diff
@@ -80,6 +85,13 @@
     public static Status fromThrowable(Throwable t) {
         t = unwrap(requireNonNull(t, "t"));
 
+        if (!exceptionMappings.isEmpty()) {
+            final Status status = exceptionMappings.get(t.getClass());
```

Comment:
Should we iterate and use `isAssignableFrom()` instead?

Reply:
👍 We need it to find the nearest exception type from the exception hierarchy.